### PR TITLE
Fix build not going red with external/flaky.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -961,14 +961,23 @@ jobs:
           name: ${{ steps.constants.outputs.external_result_name }}
           path: external-result
 
-      - name: Check
+      - name: Debug out
```
Comment (kasperl) [inline .github/workflows/ci.yml:964]: What is this? Maybe improve the name if this is something that we should look at?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -961,14 +961,23 @@ jobs:
...
+
+      - name: Check external
+        shell: bash
+        run: |
           cat external-result/*
-          ! grep "fail" external-result/*
+          grep "fail" external-result/* && exit 1
+
+      - name: Check flaky
+        if: always()
```
Comment (kasperl) [inline .github/workflows/ci.yml:976]: Can you add a comment that explains why this condition is here, but not on "Check external"?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (floitsch) [reply .github/workflows/ci.yml]: done.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (floitsch) [reply .github/workflows/ci.yml:980]: done.

**Pull request merged**